lib/record_store/provider/dynect.rb

```diff
@@ -2,6 +2,10 @@
 
 module RecordStore
   class Provider::DynECT < Provider
+    def self.record_types
+      super.add('ALIAS')
```

Comment:
We need to figure out a way for Record Store to be aware of a zone having the ability to create ALIAS records. Otherwise, people are going to get confusing error messages when attempting to add ALIAS records to unsupported zones.